server/sonar-server/src/main/java/org/sonar/server/issue/ws/SearchAction.java

```diff
@@ -178,7 +172,7 @@ public void define(WebService.NewController controller) {
       .setDescription("Comma-separated list of SCM accounts")
       .setExampleValue("torvalds@linux-foundation.org");
     action.createParam(IssueFilterParameters.ASSIGNEES)
-      .setDescription("Comma-separated list of assignee logins")
+      .setDescription("Comma-separated list of assignee logins. The value '__me__' can be used as a placeholder for the current authenticating user")
```

Comment:
Well, it depends on the viewpoint: when you are logged in on the Web UI, you are authenticated. But when you are calling the WS, you are authenticating to the WS...